RevenueCat/Scripts/PurchasesWrapperiOS.cs

```diff
@@ -93,25 +93,19 @@ public void LogOut()
         _RCLogOut();
     }
 
-    [DllImport("__Internal")]
-    private static extern void _RCCreateAlias(string newAppUserId);
     public void CreateAlias(string newAppUserId)
     {
-        _RCCreateAlias(newAppUserId);
+        // todo: no-op for now but this method should be removed
```

Comment:
can we remove it?

Reply:
I'm gonna do it in the next PR, this one just maps existing APIs to v4